### PR TITLE
Add xorg core fonts component

### PR DESCRIPTION
**Pull request by alarcher**
+ minor fixes to make-rules

### CODE REVIEW

**Comment by alarcher** [comment]
Need to check update.

**Comment by Mno-hime** [comment]
Does it add weight (as in megabytes) to the default MATE installation? If it updates: LGTM.

**Comment by alarcher** [comment]
It is just a set of packages from the X11 gate that we never migrated to oi-userland.
There is only one component left and we'll have the full gate in oi-userland (finally).

File: make-rules/shared-macros.mk [hunk 1]
```diff
@@ -378,18 +378,23 @@ MACH32 =	$(MACH32_1:i386=i86)
...
 
+BUILD_NO_ARCH =		$(BUILD_DIR_NO_ARCH)/.built
 BUILD_32 =		$(BUILD_DIR_32)/.built
 BUILD_64 =		$(BUILD_DIR_64)/.built
 BUILD_32_and_64 =	$(BUILD_32) $(BUILD_64)
+$(BUILD_DIR_NO_ARCH)/.built:	BITS=32
 $(BUILD_DIR_32)/.built:		BITS=32
 $(BUILD_DIR_64)/.built:		BITS=64
 
+INSTALL_NO_ARCH =	$(BUILD_DIR_NO_ARCH)/.installed
```
Comment (pyhalov) [inline make-rules/shared-macros.mk:397]: Please, check that these *_NO_ARCH variables don't influence setup.py NOARCH components.

Comment (alarcher) [reply make-rules/shared-macros.mk:397]: I have sucessfully rebuilt all the components in python/

**Pull request merged**